docs/tutorials/tabular_prediction/tabular-multimodal-text-others.md

```diff
@@ -0,0 +1,264 @@
+                                               eval_metric='accuracy',
+                                               stopping_metric='accuracy',
+                                               hyperparameters='default_no_hpo',
+                                               output_directory='predictor_text_multimodal')
+
+train_sentence_multimodal_embeddings = predictor_text_multimodal.extract_embedding(train_df)
+dev_sentence_multimodal_embeddings = predictor_text_multimodal.extract_embedding(dev_df)
+
+predictor_model5 = TabularPrediction.fit(train_df.join(pd.DataFrame(train_sentence_multimodal_embeddings)),
+                                         label=label_column,
+                                         time_limits=None,
+                                         eval_metric='accuracy',
+                                         stopping_metric='accuracy',
+                                         hyperparameters='default',
+                                         output_directory='model5')
+```
+
+
+```{.python .input}
+predictor_model5.leaderboard(dev_df.join(pd.DataFrame(dev_sentence_multimodal_embeddings)))
+```
+
+## Model 6: Use a larger backbone
+
+Now, we will choose to use a larger backbone: ELECTRA-base. We will find that the performance gets improved after we change to use a larger backbone model.
+
+
+```{.python .input}
+from autogluon.text.text_prediction.text_prediction import ag_text_prediction_params
+from autogluon.tabular.task.tabular_prediction.hyperparameter_configs import get_hyperparameter_config
+import copy
+
+text_nn_params = ag_text_prediction_params.create('default_no_hpo')
+text_nn_params['models']['BertForTextPredictionBasic']['search_space']['model.backbone.name'] = 'google_electra_base'
+text_nn_params['AG_args'] = {'valid_stacker': False}
+
+tabular_multimodel_hparam_v2 = copy.deepcopy(tabular_multimodel_hparam_v1)
+tabular_multimodel_hparam_v2['TEXT_NN_V1'] = text_nn_params
+
+predictor_model6 = TabularPrediction.fit(train_df,
+                                      label=label_column,
+                                      time_limits=None,
+                                      eval_metric='accuracy',
+                                      stopping_metric='accuracy',
+                                      hyperparameters=tabular_multimodel_hparam_v2,
+                                      output_directory='model6')
+```
+
+
+```{.python .input}
+predictor_model6.leaderboard(dev_df)
+```
+
+## Major Take-aways
+
+After performing these comparisons, we have the following takeaways:
+- The multimodal text neural network structure used in TextPrediction, which is based on pretrained language model, is a good network strcuture for dealing with multi-modal data.
+- K-fold bagging / stacking is helpful
+- We need a larger backbone. This aligns with the observation in recent papers, e.g., [Scaling Laws for Autoregressive Generative Modeling](https://arxiv.org/abs/2010.14701).
```

Comment:
@jwmueller How about for me to do it in a later PR (after I obtained more results).

Reply:
I can add a sentence for each model in followup PR. I think it's important to do this, otherwise this tutorial will confuse beginner users with all of the different options being presented